index.html

```diff
@@ -54,50 +65,38 @@ <h1 class="headings">ABOUT ME</h1>
     </section>
 
     <section id="portfolio">
-      <h1 class="alt-headings">PORTFOLIO</h1>
+      <h1 class="headings-first">PORTFOLIO</h1>
+      <img
+        src="./images/buddy-up.png"
+        alt="refactor project"
+        width="600"
```

Comment:
This width and height is causing the image to overflow on smaller viewports and creates a scrollbar.